apps/antalmanac/tests/calendarize-helpers.test.ts

```diff
@@ -0,0 +1,227 @@
+            prerequisiteLink: 'string',
+            section: {
+                color: 'string',
+                sectionCode: 'string',
+                sectionType: 'string',
+                sectionNum: 'string',
+                units: 'string',
+                instructors: [],
+                meetings: [
+                    {
+                        timeIsTBA: false,
+                        bldg: [],
+                        days: 'MWF',
+                        startTime: {
+                            hour: 1,
+                            minute: 2,
+                        },
+                        endTime: {
+                            hour: 3,
+                            minute: 4,
+                        },
+                    },
+                ],
+                finalExam: {
+                    examStatus: 'SCHEDULED_FINAL',
+                    dayOfWeek: 'Sun',
+                    month: 2,
+                    day: 3,
+                    startTime: {
+                        hour: 1,
+                        minute: 2,
+                    },
+                    endTime: {
+                        hour: 3,
+                        minute: 4,
+                    },
+                    bldg: [],
+                },
+                maxCapacity: 'string',
+                numCurrentlyEnrolled: {
+                    totalEnrolled: 'string',
+                    sectionEnrolled: 'string',
+                },
+                numOnWaitlist: 'string',
+                numWaitlistCap: 'string',
+                numRequested: 'string',
+                numNewOnlyReserved: 'string',
+                restrictions: 'string',
+                status: 'OPEN',
+                sectionComment: 'string',
+            },
+            term: 'string',
+        },
+    ];
+
+    const customEvents: RepeatingCustomEvent[] = [
+        {
+            title: 'title',
+            start: '01:02',
+            end: '03:04',
+            days: [true, false, true, false, true, false, true],
+            customEventID: 0,
+            color: '#000000',
+        },
+    ];
+
+    test('calendarizeCourseEvents', () => {
+        const newResult = calendarizeCourseEvents(courses);
+        const oldResult = oldCalendarizeCourseEvents(courses);
+        expect(newResult).toStrictEqual(oldResult);
+    });
+
+    test('calendarizeFinals', () => {
+        const newResult = calendarizeFinals(courses);
+        const oldResult = oldCalendarizeFinals(courses);
+        expect(newResult).toStrictEqual(oldResult);
+    });
+
+    test('calendarizeCustomEvents', () => {
+        const newResult = calendarizeCustomEvents(customEvents);
+        const oldResult = oldClendarizeCustomEvents(customEvents);
+        expect(newResult).toStrictEqual(oldResult);
+    });
+});
+
+/**
+ * TODO: Remove this and replace with an array of expected values.
+ */
+export function oldCalendarizeCourseEvents(currentCourses: ScheduleCourse[] = []): CourseEvent[] {
```

Comment:
Can you replace these with the array of expected values in this PR? Less code = better, and if you can already get these tests to pass then it should be pretty easy to just hard-code the results.

Reply:
Ok wait, actually. There's a lot more code lol. And some of the formatting is hard to capture. Like `finalExam` should be a duplicate of the original `finalExam` property, and `start` and `end` should be the original strings passed into `Date` objects. Idk, any thoughts? @EricPedley 

![image](https://github.com/icssc/AntAlmanac/assets/66761366/2435bb70-8fdd-4376-97d4-92e6a754a9f4)

Reply:
ok but that isn't more "code" per se, it's an array of hard-coded values, which is orders of magnitude easier to read than an entire other function. IMO way better than leaving an old function in there.